frameworks/mljarsupervised/exec.py

```diff
@@ -0,0 +1,84 @@
+import os
+import shutil
+import logging
+
+import pandas as pd
+import matplotlib
+from supervised.automl import AutoML
+
+from frameworks.shared.callee import call_run, result, Timer, touch
+
+matplotlib.use("agg")  # no need for tk
+log = logging.getLogger(os.path.basename(__file__))
+
+def make_subdir(name, config):
+    subdir = os.path.join(config.output_dir, name, config.name, str(config.fold))
+    touch(subdir, as_dir=True)
+    return subdir
+
```

Comment:
I added this function to `frameworks.shared.callee` as it is now used in almost every integration:
```
from frameworks.shared.callee import output_subdir
```
this will save you a few lines.

Reply:
Thanks! Changed.